src/style.css

```diff
@@ -12,6 +13,7 @@
 
 body {
   font-family: arial, sans-serif;
+  overflow: hidden;
```

Comment:
- [ ] Kindly remove the overflow hidden as it is not making it possible to see all the scores.